packages/common/styles/_colors.scss

```diff
@@ -0,0 +1,2 @@
+$charcoal: #4d575f;
+$teal: #00adbc;
```

Comment:
we still have to decide on a naming convention, so these are just placeholders right now. the goal here is to give colors a more easily-digestible name while not allowing external consumers to reference these colors directly. more explicitly, we don't want consumers to define their styles with hard-coded colors.

an alternative to this would be defining our colors as [private members](https://sass-lang.com/documentation/at-rules/use#private-members) within `_tokens.scss`. right now, having separate files feels pretty useless, but i imagine this will be convenient once we have more constants.